k8s/Makefile

```diff
@@ -6,7 +6,8 @@
 export DEPLOYMENT_ENV			?= local
 export DOMAIN      				?= ${DEPLOYMENT_ENV}.trustbloc.dev
 export BLOC_DOMAIN				?= orb-1.${DOMAIN}
-export TERM								?= xterm-256color
+export TERM						?= xterm-256color
+export PATH						?=
```

Comment:
may be you meant:
export PATH ?= ~/.local/bin:$PATH

Reply:
Updated 👍